packages/import-bundle/src/compartment-wrapper.js

```diff
@@ -42,11 +42,14 @@ export function wrapInescapableCompartment(
     // there are details to work out.
     c.globalThis.Compartment = NewCompartment;
 
-    for (const prop of Object.keys(inescapableGlobalProperties)) {
+    // Use Reflect.ownKeys, not Object.keys, because we want both
+    // string-named and symbol-named properties. Note that
+    // Reflect.ownKeys also includes non-enumerable keys.
+    for (const prop of Reflect.ownKeys(inescapableGlobalProperties)) {
       Object.defineProperty(c.globalThis, prop, {
         value: inescapableGlobalProperties[prop],
         writable: true,
-        enumerable: false,
+        enumerable: false, // TODO: really? why?
         configurable: true,
       });
```

Comment:
Should we just plainly copy the descriptor? Maybe verifying it's a data property first, but I'm not even convinced that's needed

Reply:
If we copy the descriptor, then we've potentially copied getter/setter functions written assuming a different `this`.

When we looked at this together, I at first assumed that before this PR it was `enumerable: true`, to go with `Object.keys`, in which case I was about to suggest changing to preserve the non-enumerability of newly revealed non-enumerable properties. But then when I saw that it was already `enumerable: false` for everything, I decided to leave it alone, since it is being added to a global object.

But, I suppose this should be aligned with the Compartment constructor endowment handling. Sigh. Can we make such adjustments in a distinct PR?

@kriskowal , please weigh in on Compartment constructor handling of endowed properties.

Reply:
`Compartment` uses `Object.assign` for the endowments (`globals`). If we wanted property grafting, I’d create different option `globalProperties` since `assign` is a safer default and properties are useful elsewhere. Correspondingly, `inescapableGlobals` would assign and `inescapableGlobalProperties` would graft, and this is entirely coherent.

I like @mhofman’s suggestion, with the understanding that inescapable getters and setters are sensitive to `this` binding.

Reply:
If we have 2 options, what would be the precedence of one vs the other if both define the same property? or is that an error?

Reply:
It’s a question we would have to answer.

Reply:
In a fresh environment, most properties of the global object are non-enumerable (including, I believe, all properties defined at the language level). And the list of exceptions is idiosyncratic and—outside of web browsers—rather small (e.g., Node.js has `['global', 'clearImmediate', 'setImmediate', 'clearInterval', 'clearTimeout', 'setInterval', 'setTimeout', 'queueMicrotask', 'structuredClone', 'atob', 'btoa', 'performance', 'fetch', 'crypto', '__filename', 'module', 'exports', '__dirname', 'require']`, and XS for eshost has no enumerable global properties).

I therefore think the current default is reasonable, and does not preclude a future extension that opts in to different treatment such as preserving property attributes.

Reply:
Resolved according to verbal agreement with @kriskowal stated in the extended comment in this PR's code, based on a variant of the meeting at https://www.youtube.com/watch?v=xlR21uDigGE